src/features/firstTutorial/pages/AgeInformationTutorial/AgeInformationTutorial.tsx

```diff
@@ -0,0 +1,107 @@
+
+import { AuthenticationButton } from 'features/auth/components/AuthenticationButton/AuthenticationButton'
+import { useAuthContext } from 'features/auth/context/AuthContext'
+import { UseNavigationType } from 'features/navigation/RootNavigator/types'
+import { useDepositActivationAge } from 'features/profile/helpers/useDepositActivationAge'
+import { env } from 'libs/environment'
+import { getAge } from 'shared/user/getAge'
+import { useOpacityTransition } from 'ui/animations/helpers/useOpacityTransition'
+import { ButtonQuaternarySecondary } from 'ui/components/buttons/ButtonQuarternarySecondary'
+import { ButtonWithLinearGradient } from 'ui/components/buttons/buttonWithLinearGradient/ButtonWithLinearGradient'
+import { AnimatedBlurHeaderTitle } from 'ui/components/headers/AnimatedBlurHeader'
+import { useGetHeaderHeight } from 'ui/components/headers/PageHeaderWithoutPlaceholder'
+import { InfoBanner } from 'ui/components/InfoBanner'
+import { ExternalTouchableLink } from 'ui/components/touchableLink/ExternalTouchableLink'
+import { InternalTouchableLink } from 'ui/components/touchableLink/InternalTouchableLink'
+import { ExternalSiteFilled } from 'ui/svg/icons/ExternalSiteFilled'
+import { Spacer, Typo, getSpacing } from 'ui/theme'
+import { getHeadingAttrs } from 'ui/theme/typographyAttrs/getHeadingAttrs'
+
+interface Props {
+  selectedAge?: 15 | 16 | 17 | 18
+}
+
+export const AgeInformationTutorial: FunctionComponent<Props> = ({ selectedAge }) => {
+  const { isLoggedIn, user } = useAuthContext()
+  const { goBack } = useNavigation<UseNavigationType>()
+  const { onScroll, headerTransition } = useOpacityTransition()
+  const headerHeight = useGetHeaderHeight()
+
+  const defaultAge = selectedAge ?? 15
+  const age = user?.birthDate ? getAge(user.birthDate) : defaultAge
+
+  const activationAge = useDepositActivationAge()
+  const activationText = activationAge
+    ? `Crédit activé à ${activationAge} ans`
+    : 'Crédit pas encore activé'
+
+  const headerTitle = isLoggedIn ? 'Comment ça marche\u00a0?' : `Le pass Culture à ${age} ans`
+
+  return (
+    <React.Fragment>
+      <StyledScrollView onScroll={onScroll} scrollEventThrottle={16}>
+        <Placeholder height={headerHeight} />
+        <Spacer.Column numberOfSpaces={7} />
+        <Typo.Title3 numberOfLines={3} {...getHeadingAttrs(1)}>
+          {headerTitle}
+        </Typo.Title3>
+        <Spacer.Column numberOfSpaces={6} />
+        {/* À supprimer lors de la vraie implémentation*/}
+        <Typo.Body>{activationText}</Typo.Body>
+        <Spacer.Column numberOfSpaces={4} />
+        <InfoBanner message="Cette page a-t-elle été utile&nbsp;? Aide-nous à l’améliorer en répondant à notre questionnaire.">
+          <ExternalTouchableLink
+            as={ButtonQuaternarySecondary}
+            justifyContent="flex-start"
+            icon={ExternalSiteFilled}
+            wording="Donner mon avis"
+            externalNav={{ url: env.TUTORIAL_FEEDBACK_LINK }}
+          />
```

Comment:
Il faut que le bouton est la prop inline sinon il y a un espace vide bizarre après le bouton (cf. screen) :
<img width="436" alt="Capture d’écran 2023-08-30 à 11 49 22" src="https://github.com/pass-culture/pass-culture-app-native/assets/46637324/ac132cb0-936f-41c2-bd04-7af883d8d91a">

Tu peux passer la prop inline et ajouter un spacer avant le bouton pour régler le problème :
```suggestion
          <Spacer.Column numberOfSpaces={2} />
          <ExternalTouchableLink
            as={ButtonQuaternarySecondary}
            justifyContent="flex-start"
            icon={ExternalSiteFilled}
            wording="Donner mon avis"
            externalNav={{ url: env.TUTORIAL_FEEDBACK_LINK }}
            inline
          />
```